src/main/io/gps.c

```diff
@@ -417,9 +419,40 @@ void gpsInitNmea(void)
                if (!atgmRestartDone) {
```

Comment:
`$PSRF103,00,6,00,0*23\r\n` will be sent before each custom line, without delay. Is that safe?